components/widgets/users/avatar/avatar.tsx

```diff
@@ -69,6 +69,7 @@ const Avatar = ({
             className={classes}
             alt={`${username || 'user'} profile image`}
             src={url}
+            loading='lazy'
```

Comment:
Will setting this to lazy load possibly cause the page to reflow as these load in? I can't remember if avatars have a fixed size or if that's derived from the image as its loaded?

Reply:
Based on my test it didn't cause reflows.

Reply:
They do have set widths and heights based on the size class passed.